chapter01/chapter01.adoc

```diff
@@ -0,0 +1,263 @@
+= Chapter 1: Introduction to MicroProfile
+
+== Introduction 
+
+This introductory chapter provides a comprehensive overview of the MicroProfile platform, setting the stage for subsequent chapters in this tutorial. It aims to familiarize you with the fundamentals of the MicroProfile platform, its need, and benefits. Finally, we will discuss its place in the broader context of enterprise Java development.
+
+== Topics to be covered:
+- What is MicroProfile
+- Need for MicroProfile
+- MicroProfile Specifications
+- Current MicroProfile Implementations
+- Architecture Philosophy
+- Benefits of MicroProfile
+- Relationship with Jakarta EE specification 
+
+== What is MicroProfile
+
+link:https://microprofile.io/[MicroProfile] is an open-source specification that facilitates the development of microservices using enterprise Java technologies. It defines a set of APIs and specifications for building modern, scalable, resilient, and efficient applications based on microservices architecture. The primary goal of MicroProfile is to simplify the development process for Java developers, enabling them to create applications optimized for cloud-native-development.
+
+A consortium of industry leaders, Java community members, and individual contributors collaborating within an open-source link:https://www.eclipse.org/org/workinggroups/about.php[Eclipse Foundation Working Group] developed MicroProfile. This collaborative approach ensured that MicroProfile was shaped by diverse experiences and requirements, making it a robust and versatile specification extending Java in the cloud-computing domain. 
+
+The MicroProfile Working Group currently comprises of following Java user groups and corporate members: 
+
+* Atlanta Java User Group (AJUG)
+* Association of the German Java User Groups (iJUG)
+* Garden State Java User Group (GSJUG)
+* Hunan AsiaInfo Anhui 
```

Comment:
Need to remove this one as well, because they have just left.

Reply:
Done